app.yaml

```diff
@@ -54,7 +54,7 @@ handlers:
     Cache-Control: 'public, max-age=2592000'
     Vary: Accept-Encoding
 - url: /i18n
-  static_dir: i18n
+  static_dir: assets/i18n
```

Comment:
Do we need this stanza? Can we instead change the callsites to invoke /assets/i18n instead of /i18n? I think the fewer redirections we do, the better.